rich/syntax.py

```diff
@@ -193,6 +205,24 @@ def get_background_style(self) -> Style:
         return self._background_style
 
 
+class SyntaxPosition(NamedTuple):
```

Comment:
I could have used a basic `Tuple[int, int]` for that, but I thought that some additional semantics (and reminding the user that line numbers start at 1 rather than 0 while we're here) could be nice?

Reply:
I'd probably default to a plain tuple in `SyntaxHighlightRange` and name it when you unpack.

```python
start_col, start_row = syntax_highlight_range.start
```